docs/section22/Rule22-42.md

```diff
@@ -0,0 +1,99 @@
+
+# CHW & CW - Rule 22-42  
+
+**Rule ID:** 22-42   
+**Rule Description:** The sets of performance curves specified in Table J-2 should be used to represent part-load performance of chillers in the baseline building design.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** G3.2.2.1 Baseline 
+
+**Mandatory Rule:** True  
+**Evaluation Context:** Each Chiller  
+**Table Lookup:**
+J-6
+**Function Call:**  
+
+## Applicability Check:  
+- look at each chiller - only water-cooled chillers are applicable (ie chillers with a condensing_loop): `for chiller in B_RMD.chillers:`
+    - if the chiller has a condensing_loop, and the chiller energy source is electricity, continue to rule logic: `if chiller.consensing_loop != NULL and chiller.energy_source_type == "ELECTRICITY": CONTINUE TO RULE LOGIC`
```

Comment:
should we include an applicability check for the baseline HVAC system type?

Reply:
I don't think so.  It overcomplicates things here, and if the user has issues with the HVAC system type, they'll be notified elsewhere.  If we do a check here, it just means that if a user is using the RCT to validate a model then they have to do an extra iteration.